core/matrix/dense.cpp

```diff
@@ -327,6 +328,34 @@ void Dense<ValueType>::compute_norm2_impl(LinOp *result) const
 }
 
 
+template <typename ValueType>
+void Dense<ValueType>::convert_to(Dense<ValueType> *result) const
+{
+    if (result->get_size() && result->get_size() == this->get_size()) {
```

Comment:
Did we face the issue somewhere?

Reply:
I decided to move this to a new PR: #774